opencog/nlp/learn/README

```diff
@@ -104,18 +104,16 @@ statistics.  Most of this revolves around setting up postgres.
    https://debbugs.gnu.org/cgi/bugreport.cgi?bug=26616
    (in guile-2.2, it doesn't hang, but still behaves very badly)
 
-1) Set up and configure postgres, as described in
-   `atomspace/opencog/persist/sql/README.md`
+1) Install `link-grammar-5.3.15` or newer, either from here:
 
```

Comment:
The current version of LG is 5.4.3  and contains fixes needed by other parts of the system 5.3.15 is too old.   Version 5.5 will be available in a few weeks or a month

Reply:
anyway, this is kind-of-obsoltee; the opencg repo should be build with link-grammar, so this is now a step-zero kind of thing: it has to be done before compiling opencog.